pkg/config/server/testdata/scylla-manager.yaml

```diff
@@ -46,3 +46,19 @@ repair:
   long_polling_timeout_seconds: 5
   age_max: 12h
   graceful_stop_timeout: 60s
+
+client:
+  scheme: http
```

Comment:
Scheme is https there is no way agent can run http.

Reply:
This is also an arbitrary value to make sure we can set config values. If it can only be https though then there's no reason to expose it in config at all. Will remove it as a configurable option.